yt/data_objects/profiles.py

```diff
@@ -966,7 +966,7 @@ def sanitize_field_tuple_keys(input_dict, data_source):
 
 def create_profile(data_source, bin_fields, fields, n_bins=64,
                    extrema=None, logs=None, units=None,
-                   weight_field="cell_mass",
+                   weight_field='mass',
```

Comment:
With respect to https://github.com/yt-project/yt/pull/3194 and https://github.com/yt-project/yt/pull/3101
```suggestion
                   weight_field=("gas", "mass"),
```

Reply:
I'm going to have to come back to this because @chummels felt the need to not use a tuple here because of conflicts with `yt.save_as_dataset`, see the conversation above.

Reply:
I made this change.